app/src/dataobjects/Addon.php

```diff
@@ -2,49 +2,58 @@
 
 use Elastica\Document;
 use Elastica\Type\Mapping;
+use SilverStripe\ORM\FieldType\DBDatetime;
+use SilverStripe\ORM\FieldType\DBBoolean;
+use SilverStripe\Assets\Image;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\Control\Director;
+use SilverStripe\Control\Controller;
+use SilverStripe\View\ArrayData;
+use SilverStripe\ORM\DataObject;
+
 
 /**
  * An add-on with one or more versions.
  */
 class Addon extends DataObject
 {
 
-    public static $db = array(
+    private static $db = array(
         'Name'              => 'Varchar(255)',
         'Description'       => 'Text',
         'Type'              => 'Varchar(100)',
         'Readme'            => 'HTMLText',
-        'Released'          => 'SS_Datetime',
+        'Released'          => DBDatetime::class,
```

Comment:
Was this deliberately, or an upgrader change? We can use `'Datetime'` rather than a FQCN. Same below

Reply:
Will revert, was auto-done.